SortMatrix.h

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+#include <math.h>
+using namespace std;
+inline int  bubblesort(int i, int Arr[MATRIX_SIZE][MATRIX_SIZE])
```

Comment:
Pleace, specify the criteria by which it is sorted

Reply:
fixer it boss!